src/routes/(app)/c/[channelName]/+page.svelte

```diff
@@ -28,4 +27,4 @@
     };
 </script>
 
-<InfiniteScroll init_buffer={data.posts} {get_posts} signed_in={!!data.user} />
+<InfiniteScroll init_buffer={data.posts} {getPosts} signed_in={!!data.user} />
```

Comment:
shouldn't this be `get_posts={getPosts}` since the `get_posts` prop on the `InfiniteScroll` component is still called `get_posts` for the time being? It's being renamed in another PR that has yet to be merged.